roomranger/build.gradle

```diff
@@ -2,7 +2,7 @@ plugins {
 	id 'java'
   	id 'org.springframework.boot' version '3.2.0'
 	id 'io.spring.dependency-management' version '1.1.4'
-//	id "com.github.node-gradle.node" version "7.0.1"
+	id "com.github.node-gradle.node" version "7.0.1"
```

Comment:
I feel like we don't need the node gradle for our project , so in last pull I deleted this one.